docs/objects.md

```diff
@@ -43,6 +43,28 @@ spec:
   selector:
     app: avi-server
 ```
+As `spec.loadBalancerIP` field has been deprecated from kubernetes version 1.24 onwards, AKO has introduced a new annotation `ako.vmware.com/load-balancer-ip` to L4 service that will be used to specify preferred IP. Example usage could look something like this:
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: avisvc-lb
+  namespace: red
+  ako.vmware.com/load-balancer-ip: "10.20.10.100"
```

Comment:
the annotation must be under the metadata.annotations.
for eg:
```yaml
annotations:
    ako.vmware.com/load-balancer-ip: "10.20.10.100"
```

Reply:
addressed